openshift/README.md

```diff
@@ -9,7 +9,9 @@ This vagrant-ansible script creates an setup for Heketi to manage containerized
 
 # Setup
 
-* For Libvirt type: `sudo ./up.sh --provider=libvirt`
+* For Libvirt type: `sudo ./up.sh --provider=libvirt`  
+Note: For all subsequent operations, use vagrant commands like vagrant halt and
```

Comment:
commands like `vagrant halt` and `vagrant up`..

You should use the `` ` ` which changes the font to a cli style.